### PR TITLE
[CDS-103056] CD release notes 1.64

### PR DESCRIPTION
**Pull request by vishal-av**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -47,6 +47,37 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
 ## November
 
+### Version 1.64.0
+
+#### New features and enhancements
+
+- Harness now supports **Approval APIs**, allowing users to approve or reject Harness **Manual Approval** steps using Service Account tokens. (CDS-97580, ZD-64388)
```
Comment (shylajasun) [inline release-notes/continuous-delivery.md:54]: @vishal-av can this be rephrased as Approval APIs supports Service account token for approve or reject

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -47,6 +47,37 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+- Harness now supports webhooks configured with **GitLab System Hooks** to trigger pipelines. (CDS-95423)
+
+- Harness now supports self-hosted Bitbucket OAuth tokens for the Git Experience. (CDS-95417)
+
+- The Harness platform has been updated with the Kubernetes SDK (v18.0 to v21.0) and Helm SDK (v3.12 to v3.13).
+
+- Harness now supports deployment of **Azure Functions**. (CDS-51900)
```
Comment (shylajasun) [inline release-notes/continuous-delivery.md:62]: @vishal-av this is not getting released in this release

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -47,6 +47,37 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+- Harness has introduced supports for **Canary** and **Blue-Green** Deployment strategies with Helm Charts. (CDS-35715)
```
Comment (shylajasun) [inline release-notes/continuous-delivery.md:64]: Harness has introduced native support for **Canary** and **Blue-Green** Deployment strategies with Helm Chart deployments.

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Pull request merged**